### ISSUE TITLE
Visual Studio Tooltop for Connect() is far too large.

### ISSUE THREAD

**Issue by ggppjj** [godotengine/godot-vscode-plugin#679]
### Tested versions

Reproducible in v4.3.beta2.mono.official [b75f0485b]

### System information

Windows 11- v4.3.beta2.mono.official [b75f0485b]

### Issue description

The tooltip in Visual Studio for the Connect method of a Node is... way too much. It doesn't even fit on my screen.

Example:
![image](https://github.com/godotengine/godot/assets/1223390/f0b96abc-65fc-40ff-a6da-c4d319278e28)

Because the documentation isn't _wrong_, I hope I've opened this in the right repo.

### Steps to reproduce

Attempt to connect a signal in C# using the Connect() method, hover over the Connect() method.

### Minimal reproduction project (MRP)

N/A

**Comment by dalexeev** [comment]
Is it possible to add scrolling to the popup, like in the built-in editor?

![](https://github.com/godotengine/godot-vscode-plugin/assets/47700418/97d84a74-28d6-4e75-b440-6218a63c4067)

**Comment by DaelonSuzuka** [comment]
This extension doesn't support C# at all, those tooltips aren't coming from here.

**Comment by dalexeev** [comment]
> This extension doesn't support C# at all, those tooltips aren't coming from here.

So should it be a proposal in VSCode or a C# extension for VSCode that the author uses?

**Comment by ggppjj** [comment]
This is in the source of Godot for the C# component. The tooltip is generated by Visual Studio 2022 Community, not VSCode. There is no extension, this is defined in Godot's source with comments on the method.

**Comment by dalexeev** [comment]
> There is no extension, this is defined in Godot's source with comments on the method.

Yes, C# bindings are generated from the engine API, documentation is added to C# doc comments. But they don't control whether scrollbars should be used. Yes, the `connect()` method has a rather long description, but other editors correctly display the description by detecting that the content height is too large.

<details>
<summary>Screenshots</summary>

VSCode:
![](https://github.com/godotengine/godot-vscode-plugin/assets/47700418/a09679ac-d738-4bbb-91ea-0c3c61a89d0a)

Kate:
![](https://github.com/godotengine/godot-vscode-plugin/assets/47700418/52440a76-0914-46b3-9c9a-98659aff5265)

</details>

> The tooltip is generated by Visual Studio 2022 Community, not VSCode.

I guess you should check if newer Visual Studio versions have the feature. If not, then you should suggest adding the feature to Visual Studio or some extension responsible for displaying documentation tooltips. Also, there's a chance that there's a setting "Display Scrollbars".

**Comment by ggppjj** [comment]
I apologize if this comes across as antagonistic, but Godot is the only project I've seen have such a large _summary_ section in this inline documentation. It seems unhelpful from what I understand to consider this the responsibility of the IDE, as it would seem that the summary tooltips are being used by Godot and (so far in my experience at least) by Godot alone in ways that they weren't intended or designed to support (i.e. full documentation).

The documentation on the Connect() method in particular is really the only one that I can think of even within Godot that is that large in VS2022. I just think of this more of an oversight in the code comments more than a real bug with either Godot or VS2022.

**Comment by ggppjj** [comment]
Also, would it be possible to have this issue reassigned again? I still believe the correct repo is the main Godot repo, but if nothing else I do know that the VSCode extension repo isn't right.

**Comment by DaelonSuzuka** [comment]
@dalexeev Why did you transfer this here in the first place?

**Comment by DaelonSuzuka** [comment]
> I assumed it was a bug of the plugin, but you said it doesn't support C# at all.

But this issue isn't even for the same program. Visual Studio and VSCode are totally unrelated.

**Comment by ggppjj** [comment]
> then we should probably transfer the issue to the godot-docs repository, I'll ask for an opinion in #documentation.

That would be good to my mind. I don't see a #discussion channel in either the official discord or the cafe, so I'm assuming this is an internal chat.

Thanks.

**Comment by Calinou** [comment]
> That would be good to my mind. I don't see a #discussion channel in either the official discord or the cafe, so I'm assuming this is an internal chat.

It's a public channel on the [Godot Contributors Chat](https://chat.godotengine.org/) actually :slightly_smiling_face: 

Engine development happens there, not Discord.

**Comment by ggppjj** [comment]
Ah, gotcha. Thanks!
